omps-integration.Dockerfile

```diff
@@ -0,0 +1,17 @@
+ARG OMPS_VERSION=latest
+FROM quay.io/operator-manifests/omps:${OMPS_VERSION}
+
+# Install operator-courier.
+USER 0
+RUN pip3 install koji
+COPY . /repo
+WORKDIR /repo
+RUN pip3 install -U .
+RUN rm -rf /repo
+
+# Re-install omps with this version of operator-courier.
```

Comment:
Note that the steps below are dependent on how OMPS currently sets up its image filesystem and how it is intended to be run using `gunicorn`. This might break if OMPS changes either of these things.